README.md

```diff
@@ -81,6 +81,15 @@ To contribute to IPFS in general, see the [contributing guide](https://github.co
 
 [![](https://cdn.rawgit.com/jbenet/contribute-ipfs-gif/master/img/contribute.gif)](https://github.com/ipfs/community/blob/master/contributing.md)
 
+## Releasing
+
+- Run `tx pull -a` to pull the latest translations from Transifex ([i18n#transifex-101)](https://github.com/ipfs-shipyard/i18n#transifex-101))
+- Update the version (`npm version major/minor/patch`)
+- Push the changes (`git push && git push --tags`)
```

Comment:
[`--follow-tags`](https://git-scm.com/docs/git-push/2.12.5#Documentation/git-push.txt---follow-tags)  will be safer, because it will only push tags that are reachable: 

```suggestion
- Push the changes (`git push && git push --follow-tags`)
```